shared-bindings/bitbangio/I2C.h

```diff
@@ -39,6 +39,7 @@ extern const mp_obj_type_t bitbangio_i2c_type;
 extern void shared_module_bitbangio_i2c_construct(bitbangio_i2c_obj_t *self,
     const mcu_pin_obj_t *scl,
     const mcu_pin_obj_t *sda,
+    bool internal_pullup,
```

Comment:
We don't need to remember this, I think.

Reply:
Ok, will fail on compile due to conflicting definitions? Remember that I call this method using the added param.

Reply:
Whoops, I was wrong. I thought this was part of a struct.